src/main.cpp

```diff
@@ -5247,11 +5229,6 @@ void AddResearchMagnitude(CBlockIndex* pIndex)
         stMag.interestPayments += pIndex->nInterestSubsidy;
         
         AdjustTimestamps(stMag,pIndex->nTime, pIndex->nResearchSubsidy);
-        // Track detailed payments made to each CPID
-        stMag.PaymentTimestamps         += ToString(pIndex->nTime) + ",";
-        stMag.PaymentAmountsResearch    += RoundToString(pIndex->nResearchSubsidy,2) + ",";
```

Comment:
Are you sure this field is not needed for the reward calculation?

Reply:
Yeah, it wouldn't build otherwise as the fields were removed from the struct as well.